techniques/general/G211.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Using native controls to ensure functionality is triggered on the up-event.</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Using native controls to ensure  functionality is triggered on the up-event.</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p class="instructions">Provide information below to help editors associate the technique properly. Contents of the meta section are not output by the processor.</p>
+			<p id="sc">2.5.2 Pointer Cancellation<!-- SC to which this applies --></p>
+			<p id="type">Sufficient<!-- sufficient, advisory, or failure --></p>
+		</section>
+		<section id="applicability">
+			<h2>When to Use</h2>
+			<p >This technique applies to nearly all technologies.</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>
+			<p >The objective of this technique is to ensure that users who attempt to interact with a control do not trigger the action of the event accidentally. This can be accomplished most directly by relying on the pointer's up-event (for example, the onclick or mouseup event).  </p>
+      		<p>The easiest way to meet this success criteria is simply to use the default behavior of controls and not override that behaviour with an explicit down-event trigger. The up-event is the default behaviour for almost all controls and any programming or markup language.  </p>
+     		<p>In native languages where a control is fired on the down event it is usually for good reason and is easily recoverable. For instance, an HTML input element could have the cursor enter the editable area on the “pointer down” event, because the action is trivially reversible, and as such meets the requirements of the Pointer Cancellation SC. This is because if the user realizes they made a mistake after pressing down the control, they can simply move their pointer away from the hit area while still holding down the pointer, then release their pointer and the event is not triggered. </p>
+		</section>
+
+		<section id="examples">
+			<h2>Examples</h2>
+			<section class="example">
+				<h3>Using an native onclick event in JavaScript</h3>
+				<p>In JavaScript native onclick events are triggered on the up-event by default.</p>
+			</section>
+      		<section class="example">
+				<h3>Using a native button or link in HTML</h3>
+				<p>In HTML native <code>&lt;button&gt;</code> or <code>&lt;a href ....&gt;</code> onclick events are triggered on the up-event by default.</p>
+				<p class="note"> The <code>&lt;button&gt;</code> element doesn't do anything on its own (Unless its part of a <code>&lt;form&gt;</code>, and does NOT have <code>&lt;type="button"&gt;</code>, in which case it's  submits the form). Authors should rely on input-agnostic click handlers. <p>
```

Comment:
typo and needs lowercase "(Unless its part" > "(unless it's part"; typo "in which case it's  submit the form" > "in which case it submits the form"

Reply:
Updated, thanks.